l10n/DEFAULT/CTLD.lua

```diff
@@ -209,8 +209,6 @@ ctld.dropOffZones = {
 	{ "dropzone11", "green", 2 },
 	{ "dropzone12", "green", 2 },
 	{ "dropzone13", "green", 2 },
-	{ "dropzone14", "green", 2 },
```

Comment:
What does this change do?

Reply:
For your drop points in CTLD. It defines where you can drop off the crates and will be recognized. We need to add in the dropzone 14 and 15 to include in the new dropzones at objectives. However, this broke CTLD last itme so it should be commented out/removed until we can figure out the fix.